tests/test_project_management.py

```diff
@@ -0,0 +1,222 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for firebase_admin.project_management."""
+
+import json
+
+import pytest
+
+import firebase_admin
+from firebase_admin import project_management
+from tests import testutils
+
+BASE_URL = 'https://firebase.googleapis.com'
+
+TEST_PROJECT_ID = 'hello-world'
+TEST_ANDROID_APP_ID = '1:12345678:android:deadbeef'
+TEST_ANDROID_APP_NAME = 'projects/{0}/androidApps/{1}'.format(TEST_PROJECT_ID, TEST_ANDROID_APP_ID)
+TEST_ANDROID_APP_DISPLAY_NAME = "My Android App"
+TEST_ANDROID_APP_PACKAGE_NAME = 'com.hello.world.android'
+TEST_ANDROID_APP = {
+    'name': TEST_ANDROID_APP_NAME,
+    'appId': TEST_ANDROID_APP_ID,
+    'displayName': TEST_ANDROID_APP_DISPLAY_NAME,
+    'projectId': TEST_PROJECT_ID,
+    'packageName': TEST_ANDROID_APP_PACKAGE_NAME,
+}
+TEST_ANDROID_APP_NO_DISPLAY_NAME = {
+    'name': TEST_ANDROID_APP_NAME,
+    'appId': TEST_ANDROID_APP_ID,
+    'projectId': TEST_PROJECT_ID,
+    'packageName': TEST_ANDROID_APP_PACKAGE_NAME,
+}
+
+TEST_IOS_APP_ID = '1:12345678:ios:ca5cade5'
+TEST_IOS_APP_NAME = 'projects/{0}/iosApps/{1}'.format(TEST_PROJECT_ID, TEST_IOS_APP_ID)
+TEST_IOS_APP_DISPLAY_NAME = "My iOS App"
+TEST_IOS_APP_BUNDLE_ID = 'com.hello.world.ios'
+TEST_IOS_APP = {
+    'name': TEST_IOS_APP_NAME,
+    'appId': TEST_IOS_APP_ID,
+    'displayName': TEST_IOS_APP_DISPLAY_NAME,
+    'projectId': TEST_PROJECT_ID,
+    'bundleId': TEST_IOS_APP_BUNDLE_ID,
+}
+TEST_IOS_APP_NO_DISPLAY_NAME = {
+    'name': TEST_IOS_APP_NAME,
+    'appId': TEST_IOS_APP_ID,
+    'projectId': TEST_PROJECT_ID,
+    'bundleId': TEST_IOS_APP_BUNDLE_ID,
+}
+
+OPERATION_NAME = 'operations/abcdefg'
+OPERATION_IN_PROGRESS_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': False
+})
+OPERATION_FAILED_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'error': 'some error',
+})
+ANDROID_APP_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_ANDROID_APP,
+})
+ANDROID_APP_NO_DISPLAY_NAME_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_ANDROID_APP_NO_DISPLAY_NAME,
+})
+IOS_APP_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_IOS_APP,
+})
+IOS_APP_NO_DISPLAY_NAME_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_IOS_APP_NO_DISPLAY_NAME,
+})
+
+ERROR_RESPONSE = 'some error'
+
+class TestCreateAndroidApp(object):
+    _CREATION_URL = '{0}/v1beta1/projects/{1}/{2}'.format(BASE_URL, TEST_PROJECT_ID, "androidApps")
```

Comment:
Single quotes around string literal. Here and elsewhere.

Reply:
Done.